seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -0,0 +1,90 @@
+import mock
+import responses
+
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from seqr.views.utils.terra_api_utils import get_anvil_billing_projects, get_anvil_profile, has_anvil_session,\
+    list_anvil_workspaces, get_anvil_workspace_acl
+from settings import TERRA_API_ROOT_URL
+
+
+class TerraApiUtilsCase(TestCase):
+    fixtures = ['users']
+
+    def setUp(self):
+        google_url = 'https://oauth2.googleapis.com/token'
```

Comment:
constants should be all caps and defined at the top of the file

Reply:
Okay